modules/45_carbonprice/NPi2025/datainput.gms

```diff
@@ -0,0 +1,23 @@
+*** |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/45_carbonprice/NPi2025/datainput.gms
+
+***----------------------------
+*** CO2 Tax level growing exponentially from 2025 value taken from input data
+***----------------------------
+
+pm_taxCO2eq(ttot,regi)$(ttot.val le 2025) = fm_taxCO2eqHist(ttot,regi) * sm_DptCO2_2_TDpGtC;
+
+pm_taxCO2eq(t,regi)$(t.val gt 2025) = sum(ttot, pm_taxCO2eq(ttot,regi)$(ttot.val eq 2025)) + (t.val - 2025) * (20/75) * sm_DptCO2_2_TDpGtC;
+
+loop(ext_regi$sameas(ext_regi, "EUR_regi"),
```

Comment:
Did you check this loop with @Renato-Rodrigues or someone familiar with `ext_regi`? It looks correct to me, but I am not an expert on this.

Reply:
FYI: I think there is no need to wait for an OK from one of them, but just so that Renato can have a look ex-post and adjust if necessary for the EU21 runs.